### ISSUE TITLE
update android13

### ISSUE THREAD

**Issue by pablovinicius332** [akhilkedia/AllTrans#55]
hello man, you've been missing without updating the app for new versions and I really like your app :/ any news on when a new update will come out? on a13 it doesn't work on all apps, just some google play

**Comment by amrelhossieny** [comment]
Yep same here

**Comment by zinhuofc** [comment]
Here is a log on Android 13 for those who want to fix the app, used version 1.9.3 and 1.9.5 with app Infinity for Reddit, Reddit and Harmonic that do not work
[logcat.txt](https://github.com/akhilkedia/AllTrans/files/11264851/logcat.txt)

**Comment by akhilkedia** [comment]
Updated for Android 13. Check the latest release.

**Comment by rushiranpise** [comment]
> Updated for Android 13. Check the latest release.

The scope for Settings Storage (com.android.providers.settings) gets unticked automatically in a12 oneui is this normal?

**Comment by zinhuofc** [comment]
> > Atualizado para Android 13. Verifique a versão mais recente.
> 
> O escopo do armazenamento de configurações (com.android.providers.settings) é desmarcado automaticamente em a12 oneui, isso é normal?
> 
> Obrigado pela atualização!!

mine is also unchecked, did the translation work for you?

**Comment by zinhuofc** [comment]
> > Updated for Android 13. Check the latest release.
> 
> The scope for Settings Storage (com.android.providers.settings) gets unticked automatically in a12 oneui is this normal?
> 
> Thanks for the update!!

mine is also unchecked, did the translation work for you?

**Comment by rushiranpise** [comment]
> > > Updated for Android 13. Check the latest release.
> > 
> > 
> > The scope for Settings Storage (com.android.providers.settings) gets unticked automatically in a12 oneui is this normal?
> > Thanks for the update!!
> 
> mine is also unchecked, did the translation work for you?

nope, but i notice if u reboot the device, after reboot open all trans then the app you wanted to translate then it works for a moment the again stops working
